lib/core/sessions.js

```diff
@@ -617,6 +617,21 @@ class ServerSessionPool {
   }
 }
 
+// TODO: this should be codified in command construction
+// @see https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#read-concern
+function commandSupportsReadConcern(command) {
+  return (
+    command.aggregate ||
+    command.count ||
+    command.distinct ||
+    command.find ||
+    command.mapReduce ||
```

Comment:
Do we need an additional check to ensure `mapReduce` has `out: { inline: 1 }`?

Reply:
nice catch! fixed